feugy/melodie
# feat: filter playlist dropdown menu

### What's in there?

Are included here:

- [x] feat: filter playlist dropdown menu
- [x] feat: dropdown menu know support keyboard navigation
- [x] chore: dependencies update

### How to test?

Create many playlists (more than 5), and you'll see the search box.
`AddToPlaylist` menu appears in the "Track List" drawer, in the player, and when viewing tracks of an album, in the action menu. 

### Notes to reviewers

We can not update to tailwindcss@2 yet. It implies using postcss@8, which requires webpack@5, and storybook does not support it [yet](https://github.com/storybookjs/storybook/issues/9216).
